filters/filter_21_Annoyances_Other/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 21,
+  "name": "AdGuard Other filter",
```

Comment:
I am not entirely sure about the proposed name though.

```suggestion
  "name": "AdGuard Other Annoyances filter",
```

Reply:
It's a draft name, that can be changed.